config/dev/insecure_manager.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+  labels:
```

Comment:
I moved this patch here since we're using the newer Tiltfile now. Unfortunately, both old and new Tiltfiles don't really provide an option to point to a different entry point for kustomize. I will do some more digging eventually, but for now we'll still need to manually add and remove the nonRoot param when using Tilt.

Reply:
I don't see this dev "configuration module" getting pulled in anywhere.  Is this just for posterity?

Reply:
It's intended to be an overlay. If you use `kustomize config/dev` instead of `config/default` nonRoot will be set to false. Eventually we'll want to have the tilt point here.

So, yeah kind for posterity. Basically we should figure out how to make tilt use it, but I haven't yet found a way outside of modifying the Tiltfile directly.